### PR TITLE
Temporary working the github workflow for greetings

### PR DESCRIPTION
**Pull request by yasharth291**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -4,15 +4,14 @@ on:
...
-jobs:
-  PR-Greeting:
-    name: Greeting Message to user
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/first-interaction@v1
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          pr-message: "Congratulations on making your first PR! :confetti_ball: If you haven't already, check out our [Contributing Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md) and [PR Reporting Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/PR-guidelines.md) to ensure that you are following our guidelines for contributing and creating PR."
+#jobs:
```
Comment (palisadoes) [inline .github/workflows/pull-requests.yml:7]: Yasharth. We'll still need the line

`jobs:`

**Review by palisadoes — CHANGES_REQUESTED** [review]
Yasharth. We'll still need the line

`jobs:`

File: .github/workflows/pull-requests.yml (hunk 1 above)
Comment (yasharth291) [reply .github/workflows/pull-requests.yml:7]: ok sir changing

**Comment by yasharth291** [comment]
done sir @palisadoes

**Review by palisadoes — APPROVED** [review]

**Pull request merged**